ui/src/app/model/download-artifacts.ts

```diff
@@ -0,0 +1,86 @@
+export default interface IStructuredReport {
```

Comment:
Lets rename the file to `structured-report.ts` since it only seems to be contain structs corresponding to it.